src/styles/mainStyles.js

```diff
@@ -0,0 +1,120 @@
+import styled from 'styled-components/macro';
+
+// The main styled of this project
+
+const sizes = {
```

Comment:
I will probably also start using the media q this way too now after our weeks project, it really makes the code easier :)